docs/lib-documentation.md

```diff
@@ -79,10 +91,6 @@ This class is a plain old object defining the state between two partitions (e.g.
 
 **NOTE:** These objects will often have several gRPC-specific fields inside them, but which don't need to be considered; you can construct the object however you normally instantiate objects in your language of choice (e.g. `new` in Java, `PartitionConnectionInfo{....fields...}` in Go, etc.).
 
-AvailabilityChecker
--------------------
-A class returned by [NetworkContext.addService][networkcontext_addservice] when creating a service, that provides a hook for blocking until the newly-created service is available. This allows for a more-performant workflow of 1) start many services without blocking on their availability and 2) wait for them all to become available.
-
 ### waitForStartup(Duration timeBetweenPolls, int maxNumRetries)
```

Comment:
Does this method need to be deleted as well?

Reply:
yes, I'll do it